go.mod

```diff
@@ -7,18 +7,41 @@ require (
 	github.com/gorilla/websocket v1.4.2
 	github.com/knadh/koanf v1.3.3
 	github.com/oklog/ulid/v2 v2.0.2
+	github.com/shirou/gopsutil v3.21.11+incompatible
 	github.com/stretchr/testify v1.7.0
+	go.opentelemetry.io/otel v1.3.0
```

Comment:
Might be good to have the example in a separate module to contain these dependencies.

Reply:
Yeah, I was thinking the same. We don't want the examples to pollute the dependencies of the library. I think I should introduce a separate module in a separate PR.